pkg/test/testPackages.proj

```diff
@@ -99,6 +99,10 @@
           Outputs="@(TestSupportFiles->'%(DestinationFolder)\%(FileName)%(Extension)')">
     <MakeDir Directories="%(TestSupportFiles.DestinationFolder)" />
     <Copy SourceFiles="@(TestSupportFiles)" DestinationFolder="%(TestSupportFiles.DestinationFolder)" />
+
+    <!-- Generate empty Directory.Build props/targets files so that we don't import the repo ones. -->
+    <Exec Command="echo > $(TestDir)\Directory.Build.props" />
+    <Exec Command="echo > $(TestDir)\Directory.Build.targets" />
```

Comment:
Last time I tried creating an empty Directory.Build.targets file the build failed stating it was missing the root element of the project `<Project>` tag. But in Directory.Build.props you can set a variable to disable `Directory.Build.targets` to be imported -- https://github.com/dotnet/corefx/blob/master/pkg/Microsoft.Windows.Compatibility/tests/Directory.Build.props#L3